src/login/logind-user.c

```diff
@@ -460,10 +483,12 @@ static int user_update_slice(User *u) {
 }
 
 int user_start(User *u) {
+        int r;
+
         assert(u);
 
         if (u->started && !u->stopping)
-                return 0;
+                goto service;
```

Comment:
not a fan of goto for anything but error handling... 

just invert the test and indent the following block